content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+
+@Resolution@ is made up of:
+
+- @Accuracy@ := The general priority for accuracy of location updates, used to govern any trade-off between power usage and positional accuracy. The highest positional accuracy will be achieved by specifying @Accuracy.MAXIMUM@, but at the expense of significantly increased power usage. Conversely, the lowest power usage will be achieved by specifying @Accuracy.MINIMUM@ but at the expense of significantly decreased positional accuracy.
+- @desiredInterval@ := Desired time between updates, in milliseconds. Lowering this value increases the temporal resolution. Location updates whose timestamp differs from the last captured update timestamp by less that this value are to be filtered out. Used to govern the frequency of updates requested from the underlying location provider, as well as the frequency of messages broadcast to subscribers.
+- @minimumDisplacement@ := Minimum positional granularity required, in metres. Lowering this value increases the spatial resolution. Location updates whose position differs from the last known position by a distance smaller than this value are to be filtered out. Used to configure the underlying location provider, as well as to filter the broadcast of updates to subscribers.
+
+h2(#resolution-set). Resolution set
+
+A resolution set, is a set of resolutions that must be provided by the app in order to inform the resolution policy. While developers can define their own resolution sets, Ably also provides a default resolution set that can be used for common use cases.
+
+h3(#default-resolution-set). Default resolution set
+
+The @DefaultResolutionSet@ is the set of resolutions which must be defined in order to satisfy the "default resolution constraints":#default-resolution-constraints. These are required in order to inform the "default resolution policy":#default-resolution-policy.
+
+For the @DefaultResolutionSet@ the following resolutions are expected to be included:
+
+- @farWithoutSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints), with no subscribers.
+- @farWithSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints), with one or more subscribers.
+- @nearWithoutSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints), with no subscribers.
+- @nearWithSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints), with one or more subscribers.
+
+h2(#resolution-constraints). Resolution constraints
+
+Resolution constraints specify the thresholds and corresponding logical mappings for tracking @Resolution@s that are required by the default @ResolutionPolicy@, which can be adopted by @Publisher@ instances. While developers can define their own resolution constraints, Ably also provides default resolution constraints that can be used for common use cases.
```

Comment:
> While developers can define their own resolution constraints

That's not true at the moment, we have a sealed class `ResolutionConstraints` (it's like an abstract class with a fixed number of implementations) that has only one implementation `DefaultResolutionConstraints`. This doesn't allow the users to make their own versions of `ResolutionConstraints` :thinking: @kavalerov is that how it's supposed to work?

Reply:
Waiting for input from @kavalerov

Reply:
I've removed the untrue statement. Leaving this comment unresolved for now.

Reply:
@kavalerov I don't think that there was any specific reason for sealing this class :thinking: We could turn it into an interface but then we'd need to decide how we will handle a scenario in which a user sends his custom resolution constraints to our default `ResolutionPolicy`. We can either:
1. Throw an error when a user provides any other resolution constraints than `DefaultResolutionConstraints` or
2. Ignore resolution constraints from a trackable if it isn't the `DefaultResolutionConstraints`

I think that we should go with option 1 because it will immediately inform the developer that he isn't correctly using our default `ResolutionPolicy`. If we'll go with option 2 then he won't get any feedback that he's doing something wrong and may not even notice it and get frustrated that his resolution constraints aren't working :wink: @QuintinWillison WDYT?

Reply:
Ok, on this one - I think we first need to make sure that the docs represent what we have in the code right now. (and @tbedford did this already). 

Regarding fixing this - I suggest we take this into a new issue in the ably-asset-tracking-android, and continue conversation there.

Reply:
Created https://github.com/ably/ably-asset-tracking-android/issues/299 , resolving this now.